README.md

```diff
@@ -2,13 +2,15 @@
 
 This repository contains the source for Firebase Extensions. Created and tested by Firebase, these official Firebase extensions are reliable and secure. To learn more about Firebase Extensions, including how to install them in your Firebase projects, visit the [Firebase documentation](https://firebase.google.com/docs/extensions).
 
+> New to Extensions? You can learn more at the [Firebase Extensions product page](https://firebase.google.com/products/extensions).
```

Comment:
should this link directly to marketplace? both marketplace AND firebase product page?